error/Cargo.toml

```diff
@@ -15,3 +15,7 @@ libc = "0.2"
 log = "0.4"
 serde = { version = "1.0.110", features = ["serde_derive", "rc"] }
 serde_json = "1.0.53"
+
+[package.metadata.docs.rs]
+all-features = true
+targets = ["x86_64-unknown-linux-gnu", "x86_64-apple-darwin"]
```

Comment:
Does this affect `aarch64-unknown-linux-gnu`?

Reply:
good point, will add it

Reply:
Fixed